packages/bull-vault/src/BullStrategy.sol

```diff
@@ -102,12 +125,25 @@ contract BullStrategy is ERC20, LeverageBull {
         strategyCap = _cap;
     }
 
+    /**
+     * @notice set shutdown contract that can be used to unwind the strategy if squeeth contracts are shut down
+     * @param _shutdownContract shutdown contract address
+     */
+    function setShutdownContract(address _shutdownContract) external onlyOwner {
+        require(_shutdownContract != address(0), "BS6");
+
+        emit SetShutdownContract(shutdownContract, _shutdownContract);
+
+        shutdownContract = _shutdownContract;
+    }
+
     /**
      * @notice deposit function that handle minting shares and depositing into the leverage component
      * @dev this function assume the _from depositor already have _crabAmount
      * @param _crabAmount amount of crab token
      */
     function deposit(uint256 _crabAmount) external payable {
+        require(!IController(powerTokenController).isShutDown(), "BS7");
```

Comment:
Why we have this here? but we don't in CrabV2 ?

Reply:
Also is this needed? if Controller.shutdown == true, then no one one can mint oSQTH, then can't mint new crab token, right?